test/files/run/macro-bundle-repl.scala

```diff
@@ -1,6 +1,12 @@
+import scala.tools.nsc.Settings
 import scala.tools.partest.ReplTest
 
 object Test extends ReplTest {
+    override def transformSettings(s: Settings) = {
+    s.Yreplclassbased.value = false // macros are object-based only
+    s
+  }
```

Comment:
`s.tap(_.Yreplthing = false)`. Use it or lose it.

Reply:
Good idea.  I'll switch to that on 2.13.x